bublik/urls.py

```diff
@@ -6,6 +6,7 @@
 from django.urls import include, path, re_path
 from django.views.decorators.cache import cache_page
 from rest_framework.routers import DefaultRouter
+from drf_spectacular.views import SpectacularAPIView, SpectacularRedocView, SpectacularSwaggerView
```

Comment:
The order of the import blocks is incorrect and the line is too long (will be fixed as a result of applying the linter).